dags/analyzer_helper/tests/unit/test_discord_transform_raw_members.py

```diff
@@ -0,0 +1,110 @@
+from datetime import datetime
+import unittest
+
+from dags.analyzer_helper.discord.discord_transform_raw_members import DiscordTransformRawMembers
+
+
+class TestDiscordTransformRawMembers(unittest.TestCase):
+
+    def test_transform_empty_list(self):
+        """
+        Tests that transform returns an empty list for an empty raw_members list
+        """
+        transformer = DiscordTransformRawMembers()
+        result = transformer.transform([])
+        self.assertEqual(result, [])
+
+    def test_transform_single_member(self):
+        """
+        Tests that transform correctly transforms a single member
+        """
+        transformer = DiscordTransformRawMembers()
+        raw_member = {
+            "discordId": "123456789012345678",
+            "isBot": False,
+            "joinedAt": datetime(2023, 1, 1),
+            "username": "test_user",
+            "avatar": "avatar_url",
+            "roles": ["role1", "role2"],
+            "discriminator": "1234",
+            "permissions": "some_permissions",
+            "globalName": "Global Username",
+            "nickname": "Test Nickname"
+        }
+        expected_result = {
+            "id": 123456789012345678,
+            "is_bot": False,
+            "left_at": None,
+            "joined_at": datetime(2023, 1, 1),
+            "options": {
+                "username": "test_user",
```

Comment:
Please update the expected results based on given requests on transform members comments.